include/manifold/manifold.h

```diff
@@ -430,17 +452,35 @@ class Manifold {
   static Manifold Hull(const std::vector<vec3>& pts);
   ///@}
 
+  /** @name Debugging I/O
+   * Self-contained mechanism for reading and writing high precision Manifold
+   * data.  Write functions create special-purpose OBJ files, and Read
+   * functions read them.  Be warned these are not (and not intended to be)
+   * full-featured OBJ importers/exporters.  Their primary use is as "always
+   * available" mechanisms to extract accurate mesh data for debugging
+   * purposes.  Consequently, they may store and process additional data in
+   * comments that other OBJ parsing programs won't understand.
+   *
+   * The "format" read and written by these functions is not guaranteed to be
+   * stable from release to release - it will be modified as needed to ensure
+   * it captures information needed for debugging.  The only API guarantee is
+   * that the ReadOBJ method in a given build/release will read in the output
+   * of the WriteOBJ method produced by that release.
+   */
+  static Manifold ReadOBJ(std::istream& stream);
+  static Manifold ReadOBJ(std::string& filename);
+  static Manifold ReadOBJ(const char* filename);
+  std::ostream& WriteOBJ(std::ostream& stream) const;
+  bool WriteOBJ(std::string& filename) const;
+  bool WriteOBJ(const char* filename) const;
```

Comment:
Do we need to export both `Manifold` and `MeshGL`? I'd prefer the simplicity of choosing one or the other. I'd also like to avoid any code duplication.

Reply:
Well, I guess we could go with just Manifold - my rational for adding MeshGL was to handle cases like the one I'm working on, where the input to offset is in fact not manifold.  However, I suppose strictly speaking the only thing needed for that test-wise would be reading and the code is small enough that it could just be part of the test code itself.